.github/workflows/coverage.yml

```diff
@@ -64,6 +67,7 @@ jobs:
           To this point, this PR:
             * ${(tests_changes == 0) ? "**does not** include" : "includes"} changes to tests
             * ${(docs_changes == 0) ? "**does not** include" : "includes"} changes to documentation
+            * ${process.env.WORDS_FOUND}
```

Comment:
It would be better to have something more flexible by making the string a JS object.

```js
const forbidden_words = "${process.env.WORDS_FOUND}".split(" ");
[...]
```

Then in the bullet point list:
```js
          To this point, this PR:
            * ${(tests_changes == 0) ? "**does not** include" : "includes"} changes to tests
            * ${(docs_changes == 0) ? "**does not** include" : "includes"} changes to documentation
            * ${(forbidden_words.length == 0) ? "**does not** include forbidden words" : "includes the forbidden words:" + forbidden_words.join(", ")} 
```

Reply:
That would avoid having an empty bullet point like that 

![image](https://github.com/hugsy/gef/assets/590234/cef912d6-cbd8-4524-835c-28f16b91468b)

Reply:
done